### DISCUSSION TITLE
自己打包的最新版本《瘟疫公司：进化》

### DISCUSSION THREAD

**Discussion by nukemiko** [Gamuxorg/bbs#289] [投递和请求游戏]
游戏名称：瘟疫公司：进化
游戏名称(英文)：Plague Inc: Evolved
下载地址 :（如果有的话）：https://cloud.189.cn/web/share?code=Qf2YFfFBJzae （访问码：mb63）
（加密压缩文件，密码是主站域名）

32位appimage，包含32和64位两个架构的游戏，可以直接启动，根据内核架构自动选择启动的游戏架构。
内含解药模式DLC和场景编辑器。

**Comment by ArisTocles-Nine** [comment]
芜湖谢老哥，链接在审核诶，挂了么

**Comment by nukemiko** [comment]
> 芜湖谢老哥，链接在审核诶，挂了么

链接已经更新

**Comment by hyh19962008** [comment]
请问游戏的版本号是多少啊？站里已有的版本是1.17.1

**Comment by nukemiko** [comment]
> 请问游戏的版本号是多少啊？站里已有的版本是1.17.1

版本号1.18开头，体验肯定比站内好不少，还带个解药DLC

**Comment by hyh19962008** [comment]
这个确实是重大更新，更新站里的下载链接了，感谢投稿。
有兴趣的话可以加入我们站开发者群300610898^_-

**Comment by Yamanaka-Shim-Yuki** [comment]
这个压缩包是坏了吗？为什么我输什么（github.com、github、com）都说"Corrupt file or wrong password"？
